dipper/curie_map.yaml

```diff
@@ -1,3 +1,7 @@
+# Skolemize Blank Nodes
+# overwrite iri fragment with '' for unresovable bnodes
+'_':    'https://monarchinitiative.org/.well-known/genid/'
```

Comment:
Just looking at this in RDFLIB

http://rdflib.readthedocs.io/en/stable/_modules/rdflib/term.html#BNode

     BNode.skolemize(authority='http://rdlib.net/')

        Create a URIRef “skolem” representation of the BNode, in accordance with
        http://www.w3.org/TR/rdf11-concepts/#section-skolemization

and 

http://rdflib.readthedocs.io/en/stable/apidocs/rdflib.html#rdflib.term.URIRef.de_skolemize